integrations/database/mariadb/kubearchive.sql

```diff
@@ -46,6 +46,21 @@ CREATE TABLE `resource` (
   `data` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL CHECK (json_valid(`data`)),
   PRIMARY KEY (`uuid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;
+
+--
+-- Table structure for table `log_urls`
+--
+
+DROP TABLE IF EXISTS `log_url`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `log_url` (
+  `uuid` uuid NOT NULL,
+  `url` text NOT NULL,
+  `container_name` text NOT NULL,
+  `created_at` timestamp NOT NULL DEFAULT current_timestamp(),
+  `updated_at` timestamp NOT NULL DEFAULT current_timestamp()
```

Comment:
I miss the trigger, we have one for the `resource` table but it is easily missed as the maria db statements begin with `/*!` to specify the version accepted and they "seem" comments.

The trigger is in line 75, we should add a similar one for the `log_url.updated_at` field